xml/pub/server/protocol.xml

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<protocol>
+    <struct name="DropRecord">
+        <comment>Record of an item an NPC can drop when killed</comment>
+        <field name="item_id" type="short"/>
+        <field name="min_amount" type="three"/>
+        <field name="max_amount" type="three"/>
+        <field name="rate" type="short">
+            <comment>Chance (x in 64,000) of the item being dropped</comment>
+        </field>
+    </struct>
+
+    <struct name="DropNpcRecord">
+        <comment>An NPC and the items it can drop when killed</comment>
+        <field name="npc_id" type="short"/>
+        <length name="drops_count" type="short"/>
+        <array name="drops" type="DropRecord" length="drops_count"/>
+    </struct>
+
+    <struct name="DropFile">
+        <comment>Endless Drop File</comment>
+        <field type="string" length="3">EDF</field>
+        <array name="npcs" type="DropNpcRecord"/>
+    </struct>
+
+    <struct name="InnQuestionRecord">
+        <comment>Record of a question and answer that the player must answer to register citizenship with an inn</comment>
+        <length name="question_length" type="char"/>
+        <field name="question" type="string" length="question_length"/>
+        <length name="answer_length" type="char"/>
+        <field name="answer" type="string" length="answer_length"/>
+    </struct>
+
+    <struct name="InnRecord">
+        <comment>Record of Inn data in an Endless Inn File</comment>
+        <field name="behavior_id" type="short">
+          <comment>Behavior ID of the NPC that runs the inn. 0 for default inn</comment>
+        </field>
+        <length name="name_length" type="char"/>
+        <field name="name" type="string" length="name_length"/>
+        <field name="spawn_map" type="short">
+          <comment>Map ID of the map the player is sent to after respawning</comment>
+        </field>
+        <field name="spawn_x" type="char">
+          <comment>X coordinate of the map the player is sent to after respawning</comment>
+        </field>
+        <field name="spawn_y" type="char">
+          <comment>Y coordinate of the map the player is sent to after respawning</comment>
+        </field>
+        <field name="sleep_map" type="short">
+          <comment>Map ID of the map the player is sent to after sleeping at the inn</comment>
+        </field>
+        <field name="sleep_x" type="char">
+          <comment>X coordinate of the map the player is sent to after sleeping at the inn</comment>
+        </field>
+        <field name="sleep_y" type="char">
+          <comment>Y coordinate of the map the player is sent to after sleeping at the inn</comment>
+        </field>
+        <field name="alternate_spawn_enabled" type="bool:char">
```

Comment:
The default [underlying type](https://github.com/Cirras/eo-protocol/blob/master/docs/types.md#underlying-types) of `bool` is already `char`.
`bool:char` can change to `bool`.